shared/haml/hamburger.haml

```diff
@@ -40,23 +45,32 @@
 
   show_teacher_options = "hide-always"
   show_student_options = "hide-always"
+  show_user_options = "hide-always"
   show_pegasus_options = "hide-always"
+  show_help_options = "show-always"
+  show_signed_out_options = "hide-always"
 
   hamburger_class = "none"
 
   if level
     # The header is taken over by level-related UI, so we need the hamburger
     # to show whatever would show up in the header at desktop (and mobile) widths.
     if user_type == "teacher"
+      show_user_options = "show-always"
```

Comment:
I wonder if it makes sense to just set `show_user_options = "show-always"` once if there is a level rather than set it in three places here.